### DISCUSSION TITLE
How do I change the text that shows up next to the favicon image?

### DISCUSSION THREAD

**Discussion by AkshathRaghav** [alshedivat/al-folio#2373] [Q&A] [answered]
I am asking about changing the text in these buttons that come on the browser navbar. 

![image](https://github.com/alshedivat/al-folio/assets/75845563/baa83d2a-73af-4abf-8499-0b14143ab856)

**Comment by george-gca** [comment, marked as answer]
This is the [title](https://www.w3schools.com/tags/tag_title.asp) defined in [_includes/metadata.liquid](https://github.com/alshedivat/al-folio/blob/86a6b0325959539b7b9570db081cc81acf64b963/_includes/metadata.liquid#L23)
